ShoppingCart.java

```diff
@@ -0,0 +1,232 @@
+import java.util.ArrayList;
+import java.util.Scanner;
+
+public class ShoppingCart {
+
+	/*
+	 * Object of class Item(pencil)
+	 */
+	static Item pencil = new Item("pencil", 5.0);
+	static Item eraser = new Item("eraser", 3.5);
+	static Item sharpner = new Item("sharpner", 6.9);
+	static Item scale = new Item("scale", 25.0);
+	static Item notebook = new Item("notebook", 55.5);
+
+	
+	/*
+	 * ArrayLists are for the storing of values
+	 */
+	static ArrayList<Item> cart = new ArrayList<Item>();
+
+	
+	/**
+	 * add method is used to add item to the cart.
+	 * 
+	 * @param {string}item -item is the object of class Item
+	 * @param {integer}quantity
+	 */
+	public static void add(Item item, int quantity) {
+		if (cart.contains(item)) {
+			System.out.println("This Item already exists");
+
+		} else {
+
+			if (quantity > 0) {

```

Comment:
Can define this as `else if(quantity > 0)`

Reply:
optimization done